wild/tests/integration_tests.rs

```diff
@@ -1072,10 +1100,15 @@ fn build_obj(
         }
         CompilerKind::Rust => {
             let wild = wild_path().to_str().context("Need UTF-8 path")?.to_owned();
+            let rustc_channel = match config.rustc_channel {
+                RustcChannel::Stable => "+stable",
```

Comment:
I'm wondering if we should add a 4th option - `Default` that doesn't add the argument at all. When distributions build and test wild, they likely won't have rustup installed, which I think would mean `+toolchain` wouldn't work.